appcenter-link-scripts/src/ios/index.js

```diff
@@ -39,23 +38,17 @@ module.exports = {
             console.log(`iOS App Secret is already set in ${config.plistPath}`);
             return Promise.resolve(null);
         }
-        return inquirer.prompt([{
-            type: 'input',
-            message: 'What secret does your iOS app use? [None]',
-            name: 'AppSecret',
-        }]).then((answers) => {
-            try {
-                config.set('AppSecret', answers.AppSecret);
-                return config.save()
-                    .then((file) => {
-                        console.log(`App Secret for iOS written to ${file}`);
-                        return file;
-                    });
-            } catch (e) {
-                debug('Could not save config', e);
-                return Promise.reject(e);
-            }
-        });
+        try {
+            config.set('AppSecret', 'YOUR_APP_SECRET');
+            return config.save()
+                .then((file) => {
+                    console.log(`App Secret for iOS written to ${file}`);
+                    return file;
+                });
+        } catch (e) {
```

Comment:
Not exactly related to this PR but I think this try/catch doesn't work. Probably we need to use:
```js
            return config.save()
                .then((file) => {
                    console.log(`App Secret for iOS written to ${file}`);
                    return file;
                }).catch((e) => {
                    // catch block
                });
```
Could you please check that?

Reply:
Sorry, nevermind. I've just realized that `AppCenterConfigPlist.save` before returning a `Promise` does synchronous `fs.writeFileSync` that won't be caught by `Promise.catch`.

From my perspective that's a bit unobvious design and for methods which return promises it's better to only use `Promise.reject` so we don't need any try/catch block in addition to `Promise.catch`. But refactoring is probably not in the scope of this task so this PR looks good to me.

Reply:
@roland-chernov-akvelon you have some great points:
- It's a best practice to use Promise chaining `.then().catch()` instead of mixing Promise with `try {...} catch {...}`.
- The ios `AppCenterConfigPlist.save`(https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/appcenter-link-scripts/src/ios/AppCenterConfig.js#L37) function has a problematic design of asynchronously returning Promise and synchronously throwing errors via `fs.writeFileSync();`.

As the problem is `fs.writeFileSync()` synchronously write to file system, I think we can fix this by calling the asynchronously [`fs.writeFile()`](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback).

If it's a small fix, we should just do it as part of this feature.

Reply:
@dhei Ok, we will do it as separate PR.